src/Game.java

```diff
@@ -37,11 +37,17 @@ public Game(boolean graphical) {
         Log.debug("Initializing with hexagon size: " + renderer.getPrintableHexSize());
     }
 
-    public void enterWorld() {
-        this.world = new World();
-        Log.debug("Creating world of size: (" + world.mapWidth + ", " + world.mapHeight + ")");
-        Log.debug("Initializing World Graphics...");
-        initGraphical();
+    public void enterWorld(boolean inValue) {
+        if (inValue) {
+            this.world = new World();
+            Log.debug("Creating world of size: (" + world.mapWidth + ", " + world.mapHeight + ")");
+            Log.debug("Initializing World Graphics...");
+            initGraphical();
+
+        }
+        else {
+            System.out.println("Select at least one color.");
+        }
```

Comment:
This feature can and should be done without any changes to World.java. Remember, the World class doesn't want to know about the GUI. If we were using a different GUI class that came with one selected, or chose them at random, or whatever, it should all look the same to the World class.